src/test/java/com/google/api/generator/spring/composer/goldens/EchoSpringAutoConfigurationFull.golden

```diff
@@ -132,162 +132,163 @@ public class EchoSpringAutoConfiguration {
       expandRetrySettingBuilder.setRetryDelayMultiplier(
           this.clientProperties.getExpandRetryDelayMultiplier());
-      LOGGER.info(
+      LOGGER.trace(
           "ExpandRetryDelayMultiplier set to "
               + this.clientProperties.getExpandRetryDelayMultiplier());
     }
     if (this.clientProperties.getExpandMaxRetryDelay() != null) {
       expandRetrySettingBuilder.setMaxRetryDelay(this.clientProperties.getExpandMaxRetryDelay());
-      LOGGER.info("ExpandMaxRetryDelay set to " + this.clientProperties.getExpandMaxRetryDelay());
+      LOGGER.trace("ExpandMaxRetryDelay set to " + this.clientProperties.getExpandMaxRetryDelay());
     }
     if (this.clientProperties.getExpandInitialRpcTimeout() != null) {
       expandRetrySettingBuilder.setInitialRpcTimeout(
           this.clientProperties.getExpandInitialRpcTimeout());
-      LOGGER.info(
+      LOGGER.trace(
           "ExpandInitialRpcTimeout set to " + this.clientProperties.getExpandInitialRpcTimeout());
     }
     if (this.clientProperties.getExpandRpcTimeoutMultiplier() != null) {
       expandRetrySettingBuilder.setRpcTimeoutMultiplier(
           this.clientProperties.getExpandRpcTimeoutMultiplier());
-      LOGGER.info(
+      LOGGER.trace(
           "ExpandRpcTimeoutMultiplier set to "
               + this.clientProperties.getExpandRpcTimeoutMultiplier());
     }
     if (this.clientProperties.getExpandMaxRpcTimeout() != null) {
       expandRetrySettingBuilder.setMaxRpcTimeout(this.clientProperties.getExpandMaxRpcTimeout());
-      LOGGER.info("ExpandMaxRpcTimeout set to " + this.clientProperties.getExpandMaxRpcTimeout());
+      LOGGER.trace("ExpandMaxRpcTimeout set to " + this.clientProperties.getExpandMaxRpcTimeout());
     }
     if (this.clientProperties.getExpandTotalTimeout() != null) {
       expandRetrySettingBuilder.setTotalTimeout(this.clientProperties.getExpandTotalTimeout());
-      LOGGER.info("ExpandTotalTimeout set to " + this.clientProperties.getExpandTotalTimeout());
+      LOGGER.trace("ExpandTotalTimeout set to " + this.clientProperties.getExpandTotalTimeout());
     }
     clientSettingsBuilder.expandSettings().setRetrySettings(expandRetrySettingBuilder.build());
     RetrySettings.Builder collectRetrySettingBuilder =
         clientSettingsBuilder.collectSettings().getRetrySettings().toBuilder();
     if (this.clientProperties.getCollectInitialRpcTimeout() != null) {
       collectRetrySettingBuilder.setInitialRpcTimeout(
           this.clientProperties.getCollectInitialRpcTimeout());
-      LOGGER.info(
+      LOGGER.trace(
           "CollectInitialRpcTimeout set to " + this.clientProperties.getCollectInitialRpcTimeout());
     }
     if (this.clientProperties.getCollectRpcTimeoutMultiplier() != null) {
       collectRetrySettingBuilder.setRpcTimeoutMultiplier(
           this.clientProperties.getCollectRpcTimeoutMultiplier());
-      LOGGER.info(
+      LOGGER.trace(
```

Comment:
I think adding `isTraceEnabled()` would be useful here, since we're not only doing string concatenation, but also calling a method.